design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
```

Comment:
The naming is confusing, especially when the wrapper is used for input arguments. It sounds like a set of updated prices is expected as an input.
How about `recalculatePrices`?

Please eliminate unnecessary input wrappers everywhere across the PR, see https://github.com/magento/architecture/blob/master/design-documents/graph-ql/extensibility.md

Reply:
renamed to `recalculatePrices`, removed wrapper.